test/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring.Tests/Windows/WindowsCountersTests.cs

```diff
@@ -40,7 +40,10 @@ public void WindowsCounters_Registers_Instruments()
         {
             InstrumentPublished = (instrument, listener) =>
             {
-                listener.EnableMeasurementEvents(instrument);
+                if (ReferenceEquals(meter, instrument.Meter))
```

Comment:
This also fixes a flaky test: https://dev.azure.com/dnceng-public/public/_build/results?buildId=458782&view=ms.vss-test-web.build-test-results-tab&runId=10343638&resultId=100183&paneView=debug